packages/flame/lib/src/events/interfaces/multi_tap_listener.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/gestures.dart';
+
+import '../../game/mixins/has_tappables.dart';
+import '../../game/mixins/multi_touch_tap_detector.dart';
+
+/// Interface that must be implemented by a game in order for it to be eligible
+/// to receive events from a [MultiTapGestureRecognizer].
+///
+/// Instead of implementing this class directly consider using one of the
+/// prebuilt mixins:
+///  - [HasTappables] for a `FlameGame`
+///  - [MultiTouchTapDetector] for a custom `Game`
```

Comment:
maybe I am misremembering, lmk if this comment makes sense, but can't you use MultiTouchTapDetector on `FlameGames` if you are not using tappables and want to listen to multi-touch events?

Reply:
Since FlameGame is a Game, you can absolutely use MultiTouchTapDetector on it. The HasTappables mixing is just a more concrete implementation, which ensures the event propagates down into the component tree.